plugins/woocommerce/client/legacy/js/admin/meta-boxes-order.js

```diff
@@ -1071,6 +1071,11 @@ jQuery( function ( $ ) {
 				var line_total        = $( 'input.line_total', $row );
 				var refund_line_total = $( 'input.refund_line_total', $row );
 
+				if( refund_qty > qty ) {
```

Comment:
Would it be preferable to change the quantity to the maximum allowed instead of displaying this message?

Reply:
Hmm. As written (re-capping because I realize this PR has been waiting for a while), when this is triggered:

- We show an alert reading *"Invalid line item quantity to refund."*
- However, the input value is left at the invalid value—and nothing stops the merchant from continuing.

So, if we're going to try and stop this from happening, it probably makes sense to drop the alert and set the input value to the max allowed.